Modules/Findbacio.cmake

```diff
@@ -26,6 +26,7 @@ if(DEFINED ENV{BACIO_LIB4})
   endforeach()
 endif()
 
+include(FindPackageHandleStandardArgs)
 find_package_handle_standard_args(bacio
```

Comment:
In that case, the correct usage is:
```
include (FindPackageHandleStandardArgs)
find_package_handle_standard_args (BACIO DEFAULT_MSG BACIO_LIB4 BACIO_LIB8 BACIO_LIBD)
```

Reply:
Similar changes to the other `Find{PACKAGE}.cmake`

Reply:
This doesn't export `*_LIB4` variables though. It exports an imported target the same as find-package-config.cmake

Reply:
So I just check that the path exists

Reply:
I don't think so.  What is the point of this Find{PACKAGE}.cmake if it uses the package-config as part of the cmake build.
I thought this file is to provide an instance of CMAKE-variables obtained from the environment which are a result of either a module load or user manually specifying them.

If one is using the cmake-build of the package, this file is redundant.

Reply:
It's not. It doesn't use the `find-package-config.cmake`, it just exports it in the same format.

Instead of creating the variables `BACIO_LIB4` etc. It creates an imported target from the environment variables. 

By having this export them in the same format as the `package-config.cmake` version it allows the library CMake to be able to use either the config version or this without any changes because they both create imported targets, just in a different way.

Reply:
Thanks for the explanation and clarifying my understanding.
Go ahead.